drivers/counter/timer_dtmr_cmsdk_apb.c

```diff
@@ -84,7 +84,11 @@ static int dtmr_cmsdk_apb_set_top_value(struct device *dev,
 	data->load = ticks;
 
 	/* Set the timer to count */
-	cfg->dtimer->timer1load = ticks;
+	if (reset) {
+		cfg->dtimer->timer1load = ticks;
+	} else {
+		cfg->dtimer->timer1bgload = ticks;
```

Comment:
I think some inline comment is needed here, regarding the function of the background load

Reply:
Added.

Reply:
thanks